__test__/handler.spec.ts

```diff
@@ -0,0 +1,13 @@
+// import { APIGatewayEvent, Handler, Callback, Context } from "aws-lambda";
```

Comment:
Oh! I didn't see the filename for this test. 
Most of the stuff we're writing internally at the moment has a very different directory structure.
i.e. tests sit in the same directory as the actual code files.

e.g.
```
src
├── getEnvironment.test.ts
├── getEnvironment.ts
└── index.ts
```

Would we be able to change this to fit this standard for the way we set up our tests?